### ISSUE TITLE
Fix grammar typo in importlib.reload() exception

### ISSUE THREAD

**Issue by brettcannon** [python/cpython#74048]
BPO | [29862](https://bugs.python.org/issue29862)
--- | :---
Nosy | @brettcannon, @Mariatta, @mandeep
PRs | <li>python/cpython#809</li><li>python/cpython#811</li><li>python/cpython#812</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/brettcannon'
closed_at = <Date 2017-03-25.10:47:58.230>
created_at = <Date 2017-03-20.19:08:48.390>
labels = ['3.7', 'library']
title = 'Fix grammar typo in importlib.reload() exception'
updated_at = <Date 2017-03-25.10:47:58.174>
user = 'https://github.com/brettcannon'
```

bugs.python.org fields:
```python
activity = <Date 2017-03-25.10:47:58.174>
actor = 'Mariatta'
assignee = 'brett.cannon'
closed = True
closed_date = <Date 2017-03-25.10:47:58.230>
closer = 'Mariatta'
components = ['Library (Lib)']
creation = <Date 2017-03-20.19:08:48.390>
creator = 'brett.cannon'
dependencies = []
files = []
hgrepos = []
issue_num = 29862
keywords = []
message_count = 8.0
messages = ['289901', '290074', '290099', '290239', '290464', '290474', '290475', '290476']
nosy_count = 3.0
nosy_names = ['brett.cannon', 'Mariatta', 'mandeepb']
pr_nums = ['809', '811', '812']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = None
url = 'https://bugs.python.org/issue29862'
versions = ['Python 3.5', 'Python 3.6', 'Python 3.7']
```

</p></details>

**Comment by brettcannon** [comment]
https://github.com/python/cpython/blob/05f53735c8912f8df1077e897f052571e13c3496/Lib/importlib/__init__.py#L140

"reload() argument must be a module" (missing the "a").

**Comment by aa752287-3877-4488-b28e-6a0aef4e10d3** [comment]
Hi Brett, I'd like to take this on as my first contribution to CPython if that's okay with you.

**Comment by brettcannon** [comment]
Go for it, Mandeep!

**Comment by Mariatta** [comment]
Does this need backport to 3.5/3.6?

**Comment by Mariatta** [comment]
New changeset 9f0aa4843f8c26937d5817f27cac4aae9c0a034f by Mariatta (Mandeep Bhutani) in branch 'master':
bpo-29862: Fix grammar in importlib.reload() exception (GH-809)
https://github.com/python/cpython/commit/9f0aa4843f8c26937d5817f27cac4aae9c0a034f

**Comment by Mariatta** [comment]
New changeset 8b82236952619c3838865ff535e5ce77b59b4a78 by Mariatta in branch '3.6':
bpo-29862: Fix grammar in importlib.reload() exception (GH-809) (GH-811)
https://github.com/python/cpython/commit/8b82236952619c3838865ff535e5ce77b59b4a78

**Comment by Mariatta** [comment]
New changeset 55717b4b487a9ccc119ca501fad71937fa01d1f6 by Mariatta in branch '3.5':
bpo-29862: Fix grammar in importlib.reload() exception (GH-809) (GH-812)
https://github.com/python/cpython/commit/55717b4b487a9ccc119ca501fad71937fa01d1f6

**Comment by Mariatta** [comment]
I merged Mandeep's PR and backported it into 3.5 and 3.6.
Thanks all :)
